apps/client/lib/graphql-queries.ts

```diff
@@ -1,24 +1,24 @@
 import { gql } from '@/src/__generated__'
```

Comment:
There are a lot of changes here because I fixed the tab-spacing for the queries. The only real change is that I added `getCurrentCycle`